resources/zib/zib-Pregnancy-EstimatedDateOfDelivery.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-Pregnancy-EstimatedDateOfDelivery" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy-EstimatedDateOfDelivery" />
```

Comment:
Naamgeving wijkt af van de overige profielen (- ipv .) als ik naar de andere zib 2020 profielen kijk dan lijkt het dat de . de nieuwe conventie is?